src/main/groovy/grails/mongeez/MongeezGrailsPlugin.groovy

```diff
@@ -48,9 +48,14 @@ class MongeezGrailsPlugin extends Plugin {
         log.debug "Configuring Grails Mongeez Plugin..."
 
         String mongoBeanName = 'mongo'
-        String databaseName = grailsApplication.config.grails.mongodb.databaseName
-        String username = grailsApplication.config.grails.mongodb.username
-        String password = grailsApplication.config.grails.mongodb.password
+        Map<String, Object> mongodbConfigurationMap = grailsApplication.config.grails.mongeezCreds ?:
+                grailsApplication.config.grails.mongodb
+
+        String databaseName = mongodbConfigurationMap.databaseName
+        String username = mongodbConfigurationMap.username
+        String password = mongodbConfigurationMap.password
+        String authenticationDatabase = mongodbConfigurationMap.authenticationDatabase ?:
```

Comment:
The name sounds weird `authenticationDatabase` is it coming from Mongeez?

Reply:
No Sir. I need to set `authDb` parameter here https://github.com/causecode/grails-mongeez/pull/5/files/1adf2f2280d2c4be333e9c0146a135fe8ff153b9#diff-6071aa6a15180ada13e304588b549849R77

The mongo command to log in to a protected database is `mongo -u <username> -p <password> <database> --authenticationDatabase="<db_which_keeps_authentication_records>"`

So, I used authenticationDatabase as the variable name since that was the flag in command.